src/components/scenes/CryptoExchangeScene.tsx

```diff
@@ -212,37 +212,39 @@ export class CryptoExchangeComponent extends React.Component<Props, State> {
     return false
   }
 
-  getQuote = () => {
+  onNext = () => {
```

Comment:
This should be called `handleNext`. There is a lint rule we eventually want to enable that requires this naming like `<X onFoo={handleWhatever} />` for all JSX callbacks.

Otherwise, this is a fantastic cleanup!